### DISCUSSION TITLE
How to change the icon for associated file

### DISCUSSION THREAD

**Discussion by genej101** [sumatrapdfreader/sumatrapdf#2986] [Q&A] [answered]
This was recommended to me by a Vivaldi community manager.  I like it a lot! I have one question though, is there a way to change the icon on pdf files from the big yellow banner to something more meaningful to the file?

**Comment by GitHubRulesOK** [comment, marked as answer]
pdf does not have icons it is windows applies big red blue green or yellow APPLICATION splashes so change windows settings, see lower left however there are many ways to set them to desired custom values (currently SumatraPDF uses a different one for ePUB)
![image](https://user-images.githubusercontent.com/12754913/192592554-7460e87d-6402-48d9-bf81-83f442397f8b.png)
![image](https://user-images.githubusercontent.com/12754913/192594579-e51ebef8-e586-4316-b428-edd23c009f0b.png)

**Reply by genej101** [reply]
Okay, I have that set that way already. Thanks. :^)

**Reply by GitHubRulesOK** [reply]
there are means to choose another application icon from same exe ! or other icon sources usually using a 3rd party icon manipulator much easier in win95 or xp less easy in win2K22H2
